entrypoint.sh

```diff
@@ -111,7 +111,7 @@ fi
 
 # Drain the node.
 # https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#use-kubectl-drain-to-remove-a-node-from-service
-kubectl drain ${NODE_NAME} --force --ignore-daemonsets --delete-local-data
+kubectl drain ${NODE_NAME} --force --ignore-daemonsets --delete-local-data --grace-period=120
```

Comment:
Thanks for contributing! 😁 

Adding a grace period of two minutes per pod seems optimistic when AWS gives you a total of two minutes' warning before terminating the instance.

Would be better if you make it an optional setting, read via env var. Would you mind doing that?

Reply:
Thanks for good suggestion :)

I made it env var option `GRACE_PERIOD` with default value 120

Reply:
Isn't 120 a little too high still?

Reply:
can you suggest proper value? 110 or 100?

Reply:
@egeland can you review?

Reply:
Ideally we could calculate it from the `$NOTICE_URL` request, since that'd be more precise.
Although, that endpoint seems deprecated in favor of the `instance-action` endpoint.